plugin/flags_sources/cmake_file.py

```diff
@@ -123,6 +154,7 @@ def unique_folder_name(cmake_path):
 
         Returns:
             str: Path to a unique temp folder.
+
```

Comment:
Nit: Please avoid adding additional lines where there are no direct edits of your code.